File: MyToolTrackerAPI/MyToolTrackerAPI/Models/Employee.cs
using System;
namespace MyToolTrackerAPI.Models
{
	public class Employee
	{
		public int Id { get; set; }
		public string FirstName { get; set; }
		public string LastName { get; set; }
		public string Jmbg { get; set; }
		public string IdCardNumber { get; set; }
		public string ContactNumber { get; set; }

		// Foreign keys
		public int EmployeeTypeId { get; set; }

		// Relationships
		public ICollection<OrderRequest> OrderRequests { get; set; }
		public EmployeeType EmployeeType { get; set; }
	}
}

File: MyToolTrackerClient/src/types/employee.ts
import { EmployeeType } from './employeeType';

export interface Employee {
  id: number;
  firstName: string;
  lastName: string;
  jmbg: string;
  idCardNumber: string;
  contactNumber: string;
  type: EmployeeType;   // will be changed to fetch from the server
}
